build.sbt

```diff
@@ -5,7 +5,8 @@ import Scripted._
 def baseVersion = "1.1.0-SNAPSHOT"
 def internalPath = file("internal")
 
-lazy val compilerBridgeScalaVersions = List(scala212, scala211, scala210)
+lazy val compilerBridgeScalaVersions = List(scala212, scala213, scala211, scala210)
+lazy val compilerBridgeTestScalaVersions = List(scala212, scala211, scala210)
```

Comment:
Why's 2.13 not tested?

Reply:
Some dependencies are not available for Scala 2.13.0-M2.